### ISSUE TITLE
how to get i3d pretrained model

### ISSUE THREAD

**Issue by bbp94** [naviocean/pseudo-3d-pytorch#1]
hi, @naviocean , could you provide kinetics_i3d_model_rgb.pth you used in the i3d code? Thanks very much.

**Comment by naviocean** [comment]
@bbp94 you can try it https://drive.google.com/open?id=1OnlZFmC8ra-fhp-2zHirMw6nceBTR65I

**Comment by bbp94** [comment]
@naviocean Thank you for your prompt reply, but when I click your url https://drive.google.com/open?id=1OnlZFmC8ra-fhp-2zHirMw6nceBTR65I, it reminds me that I have no permission to view it. Could you please give me a new url?

**Comment by naviocean** [comment]
@bbp94  Here you are: https://www.dropbox.com/s/58avkg5cq7oqp5l/model_rgb.pth?dl=0

**Comment by bbp94** [comment]
@naviocean Thanks for your help. I downloaded the pretrained model and used it to fintune I3D model on UCF101, but the  Loss could not decrease(4.8), and accuracy was close to 0.  Have you ever encountered this problem?

**Comment by naviocean** [comment]
@bbp94 Sorry for delay reply, I dont have time to test this i3d network and model weight. You can try another repository https://github.com/hassony2/kinetics_i3d_pytorch

**Comment by bbp94** [comment]
@naviocean That's OK. Really appreciate for your great help.

**Comment by Easyfeng222** [comment]
@bbp94 Ｅxcuse me ,Can you tell me if you successfully implemented this git?
